### ISSUE TITLE
[webpack v5] Error: Universal Chunk Loading is not implemented yet

### ISSUE THREAD

**Issue by puilp0502** [webpack/webpack#11660]
I'd like to share the ~hack~ solution for the `Error: Universal WebAssembly Loading is not implemented yet`.

This error arises from targeting both `["web", "node"]` in your webpack configuration.
Indeed, the [release log](https://webpack.js.org/blog/2020-10-10-webpack-5-release/#improved-target-option) for the webpack v5 explicitly tells us that
> ["web", "node"] will lead to an universal chunk loading method, which is not implemented yet

But you might not have the `"target"` option set. Then why the error?  
Webpack v5 also [changed defaults](https://webpack.js.org/configuration/target/#browserslist) for `"target"` option to `"browserslist"` (if `browserslist` config is available).  
Thus, if you are targeting both browsers and node in your browserslist config, you may also get this error without explicit `"target": ["web", "node"]`.

However, if you don't need chunk loading and wasm loading,  you can also use:
```js
output: {
    // ...
    chunkLoading: false,
    wasmLoading: false,
    // ...
}
```
to disable Universal Chunk Loading and Universal WebAssembly Loading.  
  
  

TL;DR: Set `output.chunkLoading` and `output.wasmLoading` to `false`.

**Comment by evilebottnawi** [comment]
Universal chunks are not implemented yet, you can find the issue using search, remove node target from browserslist or change the target option

**Comment by evilebottnawi** [comment]
Pinned for other developers

**Comment by vsphv** [comment]
I not really sure why, but after setting `chunkLoading: false`, hot module reload have the error of `Uncaught TypeError: __webpack_require__.hmrM is not a function`. Struggle on this for very long, if anyone happen to to have the same error, try remove this setting when `target: "browserslist"`.

**Comment by learner-L** [comment]
很厉害

**Comment by ckken** [comment]
chunkLoading: false, 会影响其他配置 暂时回滚

**Comment by codler** [comment]
@alexander-akait are there anywhere where we can see if the implementation is in progress?

**Comment by alexander-akait** [comment]
No progress, it is complex task with a lot of different improvements, why you need this ASAP?

**Comment by barak007** [comment]
> No progress, it is complex task with a lot of different improvements, why you need this ASAP?

As I see it. There is a huge win to use the `client` build in node SSR environment.

Is there any way this task can be broken in to a check list of what needs to be done to support this feature? maybe it will allow other to contribute.

**Comment by alexander-akait** [comment]
I think yes

/cc @sokra Can we do it?

**Comment by sibelius** [comment]
is this closed?

this issue is pinned

**Comment by alexander-akait** [comment]
`Universal` is not implement, we have other issue for this, we pin it to avoid open many of the same problems, some developers don't link to use search :smile:

**Comment by xoxefdp** [comment]
Got same problem here

Defined .browserlistrc
```
defaults
not IE 11
maintained node versions
```

Fixed changing it to
```
defaults
not IE 11
```

**Comment by ellenmartin22** [comment]
😀

**Comment by barak007** [comment]
Is there any progress on the topic?

**Comment by codeeshop-oc** [comment]
> Got same problem here
> 
> Defined .browserlistrc
> 
> ```
> defaults
> not IE 11
> maintained node versions
> ```
> 
> Fixed changing it to
> 
> ```
> defaults
> not IE 11
> ```

it works
